examples/collection-get-item/index.tsx

```diff
@@ -0,0 +1,26 @@
+import "./style.css";
+import * as Ariakit from "@ariakit/react";
+
+type CustomItem = {
+  id: string;
+  element?: HTMLElement | null | undefined;
+  color?: string;
+};
```

Comment:
Should we use `interface` to keep it consistent with component props? https://ariakit.org/guide/coding-guidelines#prefer-interface-over-type
```suggestion
interface CustomItem {
  id: string;
  element?: HTMLElement | null | undefined;
  color?: string;
}
```